<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="0" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite_1" duration-ms="447" started-at="2019-10-08T21:46:19Z" finished-at="2019-10-08T21:46:20Z">
    <groups>
    </groups>
    <test name="Validate User Login" duration-ms="105" started-at="2019-10-08T21:46:19Z" finished-at="2019-10-08T21:46:20Z">
      <class name="Mlucario.HomePage">
        <test-method status="FAIL" signature="initialize()[pri:0, instance:Mlucario.HomePage@15aab8c6]" name="initialize" is-config="true" duration-ms="0" started-at="2019-10-08T14:46:19Z" finished-at="2019-10-08T14:46:19Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at resources.base.initializeDriver(base.java:50)
	at Mlucario.HomePage.initialize(HomePage.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="SKIP" signature="testLoginFeature(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Mlucario.HomePage@15aab8c6]" name="testLoginFeature" duration-ms="78" started-at="2019-10-08T14:46:20Z" data-provider="getData" finished-at="2019-10-08T14:46:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[non_restrict_user@qw.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Non restricted user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginFeature -->
        <test-method status="SKIP" signature="testLoginFeature(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Mlucario.HomePage@15aab8c6]" name="testLoginFeature" duration-ms="1" started-at="2019-10-08T14:46:20Z" data-provider="getData" finished-at="2019-10-08T14:46:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[restricted_user@qw.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[restricted user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginFeature -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:Mlucario.HomePage@15aab8c6]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-10-08T14:46:20Z" finished-at="2019-10-08T14:46:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Mlucario.HomePage -->
    </test> <!-- Validate User Login -->
    <test name="Validate title in the page" duration-ms="4" started-at="2019-10-08T21:46:20Z" finished-at="2019-10-08T21:46:20Z">
      <class name="Mlucario.ValidateTitle">
        <test-method status="FAIL" signature="initialize()[pri:0, instance:Mlucario.ValidateTitle@5403f35f]" name="initialize" is-config="true" duration-ms="0" started-at="2019-10-08T14:46:20Z" finished-at="2019-10-08T14:46:20Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at resources.base.initializeDriver(base.java:50)
	at Mlucario.ValidateTitle.initialize(ValidateTitle.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="SKIP" signature="validateTitle()[pri:0, instance:Mlucario.ValidateTitle@5403f35f]" name="validateTitle" duration-ms="1" started-at="2019-10-08T14:46:20Z" finished-at="2019-10-08T14:46:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTitle -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:Mlucario.ValidateTitle@5403f35f]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-10-08T14:46:20Z" finished-at="2019-10-08T14:46:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Mlucario.ValidateTitle -->
    </test> <!-- Validate title in the page -->
    <test name="Validate Navigation in the page" duration-ms="4" started-at="2019-10-08T21:46:20Z" finished-at="2019-10-08T21:46:20Z">
      <class name="Mlucario.validateNavigationBar">
        <test-method status="FAIL" signature="initialize()[pri:0, instance:Mlucario.validateNavigationBar@6d4e5011]" name="initialize" is-config="true" duration-ms="0" started-at="2019-10-08T14:46:20Z" finished-at="2019-10-08T14:46:20Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at resources.base.initializeDriver(base.java:50)
	at Mlucario.validateNavigationBar.initialize(validateNavigationBar.java:17)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="SKIP" signature="validateNavigationBar()[pri:0, instance:Mlucario.validateNavigationBar@6d4e5011]" name="validateNavigationBar" duration-ms="0" started-at="2019-10-08T14:46:20Z" finished-at="2019-10-08T14:46:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateNavigationBar -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:Mlucario.validateNavigationBar@6d4e5011]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-10-08T14:46:20Z" finished-at="2019-10-08T14:46:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Mlucario.validateNavigationBar -->
    </test> <!-- Validate Navigation in the page -->
  </suite> <!-- Suite_1 -->
</testng-results>
